content/applications/inventory_and_mrp/quality/quality_check_types/instructions_check.rst

```diff
@@ -2,6 +2,7 @@
 Instructions quality check
 ==========================
 
+.. |MO| replace:: :abbr:`MO (Manufacturing Order)`
```

Comment:
I can't comment on the actual line so I'll do it here: 
In an [earlier writer workshop](https://docs.google.com/document/d/1laiWC9ow0IjXPuQp5h8L_Tj98HbE8ymGn9v-FFOwlYM/edit#heading=h.90nibixh8llr), we mentioned that when defining a concept/Odoo term for the first time, we italicize it, like how you defined `*Instructions* check` for the first time in line 8. However, in each instance of instructions quality check after, I think we can leave it without italics and in lowercase.

I also tried to communicate this in the [documentation guidelines](https://www.notion.so/Documentation-Guidelines-78bc0a486181431ebab8d4145064eafa?pvs=4#3bd4b1a077b04b138896b344b2a4f73f) we're working on together, but I still wanted to get your opinion on whether this guideline/rule is reasonable after seeing it applied in *real* documentation. 

To me, guidelines are meant to maintain consistency in writing styles among our team (not used to nitpick and enforce rules), so if you have any disagreements about a rule, I'd like to know what you're thinking! The guidelines should be flexible to what we, as a team, think makes sense :smile: